Ginger/GingerCoreNETUnitTest/LinuxTransformationTests/ActExcelTests.cs

```diff
@@ -0,0 +1,200 @@
+﻿using amdocs.ginger.GingerCoreNET;
+using Amdocs.Ginger.CoreNET.ActionsLib;
+using Amdocs.Ginger.CoreNET.Repository;
+using DocumentFormat.OpenXml.Spreadsheet;
+using GingerCore.Actions;
+using GingerCoreNETUnitTest.RunTestslib;
+using GingerTestHelper;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NPOI.HSSF.UserModel;
+using NPOI.SS.UserModel;
+using NPOI.XSSF.UserModel;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace GingerCoreNETUnitTest.LinuxTransformationTests
+{
+    [TestClass]
+    public class ActExcelTests
+    {
+        [TestInitialize]
+        public void TestInitialize()
+        {
+            WorkSpace.Init(new WorkSpaceEventHandler());
+            WorkSpace.Instance.SolutionRepository = GingerSolutionRepository.CreateGingerSolutionRepository();
+        }
+        
+        [TestMethod]
+        public void ReadExcelFirstRowTest()
+        {
+            //Arrange            
+            ActExcel actExcel = new ActExcel();
+            actExcel.AddOrUpdateInputParamValueAndCalculatedValue("ExcelFileName",
+                TestResources.GetTestResourcesFile(@"Excel" + Path.DirectorySeparatorChar + "Names.xlsx"));
+            actExcel.AddOrUpdateInputParamValueAndCalculatedValue("SheetName","Sheet1");
+            actExcel.ExcelActionType = ActExcel.eExcelActionType.ReadData;
+            actExcel.AddNewReturnParams = true;
+
+            //Act
+            actExcel.Execute();
+
+            //Assert
+            Assert.AreEqual(actExcel.ActReturnValues.Count,4);
```

Comment:
Add validation of actual returned values for all unit tests

Reply:
done